vg_chart_generator/vg_chart_generator/__init__.py

```diff
@@ -0,0 +1,63 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import argparse
+
```

Comment:
What does argparse do? I am guessing it means argument parser. but I don't know what the library is used for.